src/paper/ingestion/clients/arxiv.py

```diff
@@ -154,6 +158,12 @@ def fetch_recent(
         if since is None:
             since = until - timedelta(days=7)
 
+        # ArXiv API requires at least 3 days between since and until dates.
+        min_days_required = 3
```

Comment:
This is OK for now but what would be really nice is have a declarative structure with config options as such. 

```
since = until - timedelta(days=7)
```

```
min_days_required
```

Can go in such config

Reply:
We already have provider-specific configuration classes for this purpose, e.g., https://github.com/ResearchHub/researchhub-backend/blob/main/src/paper/ingestion/clients/arxiv.py#L20.
Please note that this is not a configuration option, but a specific restriction of the provider that is not configurable from our side (and will also not change across environments). The number defines the minimum delta in days that the arXiv API requires to yield results. I assign it to a variable name here merely to avoid any "magic numbers" in our sources.